apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/EnumMap.cs

```diff
@@ -29,6 +29,17 @@ public ApiValueAttribute(string value)
         {
             Value = value;
         }
+
+        internal static void ForEachStaticField<T>(Action<T, string> execute)
```

Comment:
Rather than a ForEach method, why not write a method returning `IEnumerable<KeyValuePair<T, string>>`? (That could just use LINQ, by the looks of it.)

Then the methods below that use ForEach can just use ToDictionary instead.

Reply:
Oh yes, that's nice. Doing.